pkg/cinder/const.go

```diff
@@ -65,8 +65,8 @@ const (
 	// Cinder is the global ServiceType that refers to all the components deployed
 	// by the cinder operator
 	Cinder storage.PropagationType = "Cinder"
-	//LogSnippet -
-	LogSnippet = "[DEFAULT]\nlog_file="
```

Comment:
This seemed more a hack that helped for the cinder repo tagging for dev preview.
I removed the `LogSnippet` with this patch as services should be able to have this information in the `00-config.conf` template itself and at least they should be able to provide the path value.